API/Services/ArchiveService.cs

```diff
@@ -328,12 +328,11 @@ public bool IsValidArchive(string archivePath)
         return false;
     }
 
-    private static bool ValidComicInfoArchiveEntry(string fullName, string name)
+    private static bool BackupComicInfoArchiveEntry(string fullName, string name)
```

Comment:
The method name is not very clear, it should be IsComicInfoArchiveEntry() which indicated that it's checking an entry to be valid and in the caller, we see the context of the backup logic.